src/plugins/i18n.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Daniel Erat and Niniane Wang. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. 
+
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+
+Vue.use(VueI18n);
+
+const messages = {
+  'en': {
```

Comment:
optional for later: omit quotes on object properties

Reply:
Ok, omitted quotes.